docs/topics/whatsnew-eap.md

```diff
@@ -921,6 +1004,18 @@ When configuring your project, keep in mind that some old ways of setting up com
 We encourage you to try the new DSL out in your multiplatform projects and leave feedback in [YouTrack](https://kotl.in/issue),
 as we plan to make this DSL the recommended approach for configuring compiler options.
 
+### New Compose compiler Gradle plugin
+
+The Jetpack Compose compiler, which translates composables into Kotlin code, has now been merged into the Kotlin repository.
+This will help transition Compose projects to Kotlin 2.0.0, as the Compose compiler will always ship simultaneously with
```

Comment:
```suggestion
This will help transition of Compose projects to Kotlin 2.0.0, as the Compose compiler will always be released simultaneously with
```